custom/icds_reports/utils/data_pull/governance_dashboard.sql

```diff
@@ -0,0 +1,185 @@
+    SUM(CASE WHEN age_tranche::INTEGER<=36 and sex='F' THEN valid_all_registered_in_month else 0 END) as F_open_register_till_month_0_3,
+    SUM(CASE WHEN age_tranche::INTEGER between 37 and 72 and sex='F' THEN valid_all_registered_in_month else 0 END) as F_open_register_till_month_3_6,
+
+    SUM(CASE WHEN age_tranche::INTEGER<=36 and sex='M' THEN valid_in_month else 0 END) as M_open_valid_till_month_0_3,
+    SUM(CASE WHEN age_tranche::INTEGER between 37 and 72 and sex='M' THEN valid_in_month else 0 END) as M_open_valid_till_month_3_6,
+    SUM(CASE WHEN age_tranche::INTEGER<=36 and sex='M' THEN valid_all_registered_in_month else 0 END) as M_open_register_till_month_0_3,
+    SUM(CASE WHEN age_tranche::INTEGER between 37 and 72 and sex='M' THEN valid_all_registered_in_month else 0 END) as M_open_register_till_month_3_6
+
+
+from "child_health_monthly" child_health where month='2019-10-01'
+group by awc_id;
+/*
+ HashAggregate  (cost=0.00..0.00 rows=0 width=0)
+   Group Key: remote_scan.awc_id
+   ->  Custom Scan (Citus Real-Time)  (cost=0.00..0.00 rows=0 width=0)
+         Task Count: 64
+         Tasks Shown: One of 64
+         ->  Task
+               Node: host=100.71.184.232 port=6432 dbname=icds_ucr
+               ->  Finalize GroupAggregate  (cost=463779.67..469225.78 rows=8043 width=65)
+                     Group Key: awc_id
+                     ->  Gather Merge  (cost=463779.67..468642.66 rows=40215 width=65)
+                           Workers Planned: 5
+                           ->  Sort  (cost=462779.59..462799.70 rows=8043 width=65)
+                                 Sort Key: awc_id
+                                 ->  Partial HashAggregate  (cost=462177.43..462257.86 rows=8043 width=65)
+                                       Group Key: awc_id
+                                       ->  Parallel Index Scan using chm_month_supervisor_id_102648 on child_health_monthly_102648 child_health  (cost=0.56..454458.92 rows=134235 width=43)
+                                             Index Cond: (month = '2019-10-01'::date)
+(17 rows)
+ */
+
+
+COPY(select
+state_name,
+district_name,
+block_name,
+supervisor_name,
+awc_name,
+sum(F_open_valid_till_month_0_3) as "F_open_valid_till_month_0_3",
+sum(F_open_valid_till_month_3_6) as "F_open_valid_till_month_3_6",
+sum(F_open_register_till_month_0_3) as "F_open_register_till_month_0_3",
+sum(F_open_register_till_month_3_6)as "F_open_register_till_month_3_6",
+
+sum(M_open_valid_till_month_0_3) as "M_open_valid_till_month_0_3",
+sum(M_open_valid_till_month_3_6) as "M_open_valid_till_month_3_6",
+sum(M_open_register_till_month_0_3) as "M_open_register_till_month_0_3",
+sum(M_open_register_till_month_3_6)as "M_open_register_till_month_3_6"
+from agg_awc_monthly left join
+    temp_child_data_pull_oct t on agg_awc_monthly.awc_id=t.awc_id
+where aggregation_level=5 and agg_awc_monthly.month='2019-11-01' and num_launched_awcs=1
```

Comment:
used  agg_awc_monthly.month='2019-11-01'  even for Nov so as to consider latest list of awcs launched